task1/src/components/container/conntainer.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import SearchBar from "../search/search_bar.js";
+import List from "../list/list.js";
+import Pagination from "../pagination/pagination-btn.js";
+
+const API_KEY = process.env.REACT_APP_API_KEY;
+
+export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      query: "",
+      url: `https://www.googleapis.com/youtube/v3/search?key=${API_KEY}&type=video&part=snippet&maxResults=15&q=`,
+      ytApiData: [],
+      nextPageToken: "",
+      prevPageToken: "",
+    };
+    this.handleQueryState = this.handleQueryState.bind(this);
+    this.setStateNextToken = this.setStateNextToken.bind(this);
+    this.setStatePrevToken = this.setStatePrevToken.bind(this);
+  }
+  setStateNextToken() {
+    this.setState({
+      url: `https://www.googleapis.com/youtube/v3/search?key=${API_KEY}&type=video&part=snippet&maxResults=15&q=${this.state.query}&pageToken=${this.state.nextPageToken}`,
+    });
```

Comment:
```suggestion
    this.setState({
      ...state,
      url: `https://www.googleapis.com/youtube/v3/search?key=${API_KEY}&type=video&part=snippet&maxResults=15&q=${this.state.query}&pageToken=${this.state.nextPageToken}`,
    });
```

Reply:
Once url property is removed from the store, only update the intended property. It is nextPageToken for this change